DESIGN.md

```diff
@@ -12,9 +12,9 @@ With some exceptions, the `tune` command calculates settings for each service ba
 
 The ratio, minimum, and maximum vary based upon the PE Infrastructure type and the PE services sharing resources on each PE Infrastructure host.
 
-The supported minimum system resources for the `tune` command are 4 CPU / 8 GB RAM.
+The supported minimum system resources of the Master for the `tune` command is 4 CPU / 8 GB RAM.
```

Comment:
This change "of the Master" is not true: `meets_minimum_system_requirements?` is executed for all hosts.

But I noticed that we warn now rather than exit, and the minimum is now 2 CPU / 6144 GB RAM (our VMPoolers).

Reply:
You're right.  I thought compilers could go smaller, but I think I was looking at old docs.  I think it is okay to stay with the 4/8 since that is "supported".  The use with VM's isn't technically supported for customers.

Reply:
Supporting 2/6 is designed to allow `pe_acceptance_tests` to test the command.
But at that level, we are splitting hairs, with hairs being system resources.